packages/ref-core/src/cmip_ref_core/dataset_registry/__init__.py

```diff
@@ -14,7 +14,7 @@
 import pooch
 from loguru import logger
 
-DATA_VERSION = "v3.9"
+PMP_DATA_VERSION = "v3.9"
```

Comment:
This could benefit from a more plugin-like architecture, where the cmip_ref_core package provides the infrastructure and the actual content is provided by the metrics packages.

Reply:
This PR got a bit big with that change, but this adds a lot more consistency across the packages